python/google/protobuf/internal/json_format_test.py

```diff
@@ -202,6 +202,17 @@ def testExtensionToDictAndBack(self):
     json_format.ParseDict(message_dict, parsed_message)
     self.assertEqual(message, parsed_message)
 
+  def testExtensionToDictAndBackWithScalar(self):
+    message = unittest_mset_pb2.TestEmptyMessageContainer()
+    ext1 = unittest_mset_pb2.TestDummyMessageWithExtension.scalar_extension
```

Comment:
Test show TestDummyMessageWithExtension is not defined.
You could use TestAllExtensions for test
https://github.com/protocolbuffers/protobuf/blob/d9ccd0c0e6bbda9bf4476088eeb46b02d7dcd327/src/google/protobuf/unittest.proto#L221